src/pages/relicans.js

```diff
@@ -0,0 +1,347 @@
+import React from 'react';
+import cx from 'classnames';
+import SEO from '../components/Seo';
+import PageLayout from '../components/PageLayout';
+import FeatherIcon from '../components/FeatherIcon';
+import styles from './relicans.module.scss';
+
+import aaronAvatar from '../images/relicans/aaron-bassett.jpg';
+import aishaAvatar from '../images/relicans/aisha-blake.jpg';
+import aliAvatar from '../images/relicans/ali-finkelstein.jpg';
+import chrisAvatar from '../images/relicans/chris-dabatos.jpg';
+import danielAvatar from '../images/relicans/daniel-kim.jpg';
+import dannyAvatar from '../images/relicans/danny-ramos.jpg';
+import jonanAvatar from '../images/relicans/jonan-scheffler.jpg';
+import kirkAvatar from '../images/relicans/kirk-haines.jpg';
+import miaAvatar from '../images/relicans/mia-moore.jpg';
+import pachiAvatar from '../images/relicans/pachi-carlson.jpg';
+import rachaelAvatar from '../images/relicans/rachael-wright-munn.jpg';
+
+const RelicansPage = () => {
+
+  const teamMembers = [
```

Comment:
It be cool if all the team member social icons were positioned in the same location regardless of the <p> size. It looks a little odd to see these icons positioned in different locations on the page. 

<img width="1340" alt="Screen Shot 2021-01-27 at 9 12 43 AM" src="https://user-images.githubusercontent.com/1395158/106019403-da07f600-607f-11eb-90a0-d6b2e30b0ab4.png">

Reply:
It's difficult with the Bios all being such different lengths. Maybe if I moved them before the bio?